components/script/dom/activation.rs

```diff
@@ -99,13 +88,14 @@ pub fn synthetic_click_activation(
         None,
         None,
     );
+    //Step 5
     let event = mouse.upcast::<Event>();
-    if source == ActivationSource::FromClick {
+    if not_trusted {
         event.set_trusted(false);
     }
+    //Step 9
     target.dispatch_event(event);
-
-    // Step 5
+    event.dispatching();
```

Comment:
this does nothing, please remove this line

the spec asks us to return the result of dispatching, that would mean returning the `EventStatus` returned  by `dispatch_event` (however I don't see anything in the spec that uses this right now)